pkg_resources/__init__.py

```diff
@@ -68,13 +69,9 @@
 from os import open as os_open
 from os.path import isdir, split
 
-try:
-    import importlib.machinery as importlib_machinery
 
-    # access attribute to force import under delayed import mechanisms.
-    importlib_machinery.__name__
-except ImportError:
-    importlib_machinery = None
+# access attribute to force import under delayed import mechanisms.
+importlib.machinery.__name__
```

Comment:
idk how much import order matters for this one, so I left it right there. But I'd prefer placing it after regular top-level imports if possible.

My guess is that was just to trigger the `ImportError`. In which case this line can probably be removed entirely.

```suggestion
```